code/process-GLEIF-daily.sh

```diff
@@ -93,9 +93,9 @@ zip upload.zip L1Data.ttl L2Data.ttl RepExData.ttl
 # Upload to data.world
 echo uploading to data.world dataset $1
 curl -H "Authorization: Bearer $DATAWORLD_TOKEN" \
-  -X PUT -H "Content-Type: application/octet-stream" \
-  --data-binary @upload.zip \
-  https://api.data.world/v0/uploads/$1/files/upload.zip?expandArchive=true
+  -X POST -H "Content-Type: multipart/form-data" \
+  --form file='@upload.zip' \
```

Comment:
I forget, when you specify file does it still need the @

Reply:
it does not, in fact it does NOT work with it.

Reply:
wait, sorry, misspoke - yes you DO need it.

Reply:
I actually tried this API out locally to validate it (not on the GLEIF data, on a small toy zip file) but to ensure that this curl command does work properly.